ts/screens/profile/PrivacyMainScreen.tsx

```diff
@@ -58,96 +43,121 @@ const getRequestProcessingAlertSubtitle = () => ({
   DELETE: I18n.t("profile.main.privacy.removeAccount.alert.oldRequestSubtitle")
 });
 
-class PrivacyMainScreen extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = { requestProcess: false };
-  }
+/**
+ * A screen to show the main screen of the Privacy section.
+ * Here the user can:
+ * - display the accepted privacy policy (or Term of Service)
+ * - send a request to delete his profile
+ * - send a request to export all his data
+ */
+const PrivacyMainScreen = ({ navigation }: Props) => {
+  const dispatch = useIODispatch();
+  const userDataProcessing = useSelector(userDataProcessingSelector);
```

Comment:
What do you think about using `useIOSelector`?

Reply:
Changed https://github.com/pagopa/io-app/pull/4271/commits/a6fb4f5471c7c55c5ab02a34e135c56dd19d0758